File: check_db.py
import os
from dotenv import load_dotenv
from supabase import create_client
load_dotenv("e:/talk/.env")

url = os.environ["SUPABASE_URL"]
key = os.environ["SUPABASE_SERVICE_ROLE"]
sb = create_client(url, key)

# Get all records for shop_id=8
res = sb.table("dialogs").select("id, shop_id, created_at, score, dialog_index").eq("shop_id", 8).order("created_at", desc=True).limit(30).execute()
print(f"Records for shop_id=8 (last 30):")
for r in res.data:
    print(f"  id={r['id']} | date={r['created_at']} | score={r['score']} | idx={r['dialog_index']}")
print(f"\nTotal returned: {len(res.data)}")

# Also check ALL shop_ids to see what's there
res2 = sb.table("dialogs").select("shop_id, created_at").order("created_at", desc=True).limit(10).execute()
print(f"\nLatest 10 records (all shops):")
for r in res2.data:
    print(f"  shop_id={r['shop_id']} | date={r['created_at']}")

File: dashboard/check_db.js
const {createClient} = require('@supabase/supabase-js');
const s = createClient(
  'https://itllqtmuvktatmpalzxo.supabase.co',
  'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Iml0bGxxdG11dmt0YXRtcGFsenhvIiwicm9sZSI6InNlcnZpY2Vfcm9sZSIsImlhdCI6MTc3OTI1ODY2NSwiZXhwIjoyMDk0ODM0NjY1fQ.LRVO8vAD-vOBmGI96JqIMkldk9eWDzqrXxlyTOkUIZQ'
);

(async () => {
  const {data: shops, error: se} = await s.from('shops').select('*');
  console.log('Shops:', shops);
  if (se) console.log('Shops error:', se);

  const {data: dialogs, error: de, count} = await s.from('dialogs').select('id, shop_id, score, created_at', {count: 'exact'});
  console.log('\nDialogs count:', count);
  if (dialogs) dialogs.forEach(d => console.log(`  id=${d.id} shop=${d.shop_id} score=${d.score} date=${d.created_at}`));
  if (de) console.log('Dialogs error:', de);

  const {data: tel} = await s.from('agent_telemetry').select('*');
  console.log('\nTelemetry:', tel);
})();
